spec/abstract/RISCV64/ArchInterrupt_A.thy

```diff
@@ -25,6 +25,10 @@ definition arch_mask_irq_signal :: "irq \<Rightarrow> (unit,'z::state_ext) s_mon
   where
   "arch_mask_irq_signal irq \<equiv> return ()"
 
+definition handle_spurious_irq :: "(unit,'z::state_ext) s_monad"
+  where
```

Comment:
style: should go in one line with `definition`

Reply:
In this one, it would actually make the style in this file inconsistent (the rest consistently uses the old style). Happy to change it, but I though we wanted to keep consistent where possible.